packages/snap-preact-components/src/components/Molecules/Carousel/readme.md

```diff
@@ -17,28 +17,28 @@ Additional [Swiper Modules](https://swiperjs.com/swiper-api#modules) can be prov
 The `loop` prop enables 'infinite' looping through the result set when swiping or using the arrow buttons.
 
 ```jsx
-<Carousel loop>{children}</Carousel>
+<Carousel loop={true}>{children}</Carousel>
```

Comment:
Just by existing these props are 'truthy', but this is definitely more explicit so a nice change.